report/assets/styles.css

```diff
@@ -72,8 +72,8 @@ h4 {
         font-size: 20pt;
 }
     .tableplay {
-        width:70%;
-        font-size: 20pt;
+        width:50%;
+        font-size: 12pt;
         margin-top:15px;
         margin-bottom: 2cm;
         margin-left:auto;
```

Comment:
This `margin-left` seems useless to me, when I take it out it displays the table in a nicer way

Reply:
But then the tables are not aligned in the center of the page anymore if we take the auto margins away

Reply:
I think there are other better ways to center the objects, with align-items: center or so (for media: screen)